template.md

```diff
@@ -0,0 +1,24 @@
+:x: DCO check failed
+
+This project uses the [Developer Certificate of Origin](https://github.com/probot/dco/#how-it-works) (DCO), which is a lightweight way for you to certify that you wrote or otherwise have the right to submit the code you are contributing to this project.
+
+If you agree with the DCO, add a `Signed-off-by: You Name <your@email.com>` line to each of your commits. You can easily add this with:
+
+```
+$ git rebase --signoff {{ base }}
+```
+
+After adding the DCO signoff to your commits, you will be required to force push your changes to this branch.
+
+```
+$ git push --force
```

Comment:
Would it be necessary to make this `git push --force origin {{ branch-name }}` ?

Reply:
Good call. The remote name will depend on how they cloned, but I think that's right for the common case.

Reply:
Yea I thought about that too, but I don't think we could figure that out. We could maybe add some text right below explaining how that would work?